### PR TITLE
Feat/CRN-635 cleanup alerts and notifications

### PR DESCRIPTION
**Pull request by niezgoda**
# Requirements
- Create a new channel client-asap-notifications (done)
- Move all the notifications from client-asap-engineers to the above channel.  (done)
- Move successful master build notifications from client-asap-alerts to client-asap-notifications (from Gitlab)
- Fix Github actions failure alerts

### CODE REVIEW

File: .github/workflows/env-teardown.yml [hunk 1]
```diff
@@ -77,8 +77,9 @@ jobs:
           echo "ALGOLIA_API_KEY=$(aws ssm get-parameter --name 'algolia-api-key-ci-dev' --query Parameter.Value --output text)" >> $GITHUB_ENV
       - name: Delete the index
         run: |
+          exit 1
```
Comment (kmaid) [inline .github/workflows/env-teardown.yml:80]: Feels like this wont work as designed

File: .github/workflows/env-teardown.yml [hunk 2]
```diff
@@ -78,7 +78,7 @@ jobs:
       - name: Delete the index
         run: |
           cd apps/asap-server
-          export ALGOLIA_INDEX=asap-hub_CI-${{ needs.setup.outputs.pr-number }}
+          export ALGOLIA_INDEX=asap-hub_research_outputs_CI-${{ needs.setup.outputs.pr-number }}
```
Comment (kmaid) [inline .github/workflows/env-teardown.yml:81]: This feels like a bad merge

File: .github/actions/slack/action.yml [hunk 3]
```diff
@@ -17,4 +17,4 @@ runs:
         SLACK_MESSAGE: ${{ inputs.message }}
         SLACK_ICON: https://asap-misc.s3.eu-west-1.amazonaws.com/robot.png
         SLACK_USERNAME: ASAP robot
-        SLACK_COLOR: ${{ job.status }}
+        SLACK_COLOR: ${{ inputs.color || job.status }}
```
Comment (peterstarling) [inline .github/actions/slack/action.yml:20]: wasnt the point of this to resolve the color from the status? like if we just g ive it a color as input surely it wont depend on the outcome of the previous job no?

**Review by peterstarling — CHANGES_REQUESTED** [review]

File: .github/workflows/env-teardown.yml [hunk 4]
```diff
@@ -53,7 +53,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Rebuild dependencies
-        run: yarn rebuild
+        run: exit 1 && yarn rebuild
```
Comment (markcsims) [inline .github/workflows/env-teardown.yml:56]: Need to remove this.

File: .github/workflows/env-teardown.yml [hunk 5]
```diff
@@ -73,6 +73,7 @@ jobs:
         uses: actions/checkout@v2
       - name: Fetch the Algolia keys
         run: |
+          exit 1
```
Comment (markcsims) [inline .github/workflows/env-teardown.yml:76]: Need to remove this.

File: .github/actions/slack/action.yml [hunk 6]
```diff
@@ -13,8 +13,8 @@ runs:
     - name: Slack Notification
       uses: rtCamp/action-slack-notify@v2
       env:
-        SLACK_WEBHOOK: ${{ inputs.webhook }}
+        SLACK_WEBHOOK: ${{ inputs.webhook }} - with status ${{ job.status }}
         SLACK_MESSAGE: ${{ inputs.message }}
         SLACK_ICON: https://asap-misc.s3.eu-west-1.amazonaws.com/robot.png
         SLACK_USERNAME: ASAP robot
-        SLACK_COLOR: ${{ job.status }}
+        SLACK_COLOR: ${{ (job.status == 'success' && '#00FF00') || '#FF0000' }}
```
Comment (markcsims) [inline .github/actions/slack/action.yml:20]: The status can be success, failure or cancelled. Should cancelled show up as the same colour as failed?

**Review by markcsims — CHANGES_REQUESTED** [review]

File: .github/workflows/env-teardown.yml (hunk 4 above)
Comment (niezgoda) [reply .github/workflows/env-teardown.yml:56]: this task is in WIP :)

**Pull request closed without merging**